ci/test/pipeline.template.yml

```diff
@@ -43,6 +43,20 @@ steps:
         agents:
           queue: builder-linux-x86_64
 
+      - id: rust-upload-debug-symbols-x86_64
+        label: ":rust: Upload debug symbols for x86_64"
```

Comment:
Do you mind dropping the ":rust:" emoji? I've used it to keep track of pipelines that run with Cargo and not Bazel, but this just uses some command line tools in the builder image.

Reply:
Will do